PyExpLabSys/apps/keithley_2450_probe_station/probestation.py

```diff
@@ -0,0 +1,482 @@
+from PyQt5 import QtWidgets, QtCore, uic
+from PyQt5.QtWidgets import QMessageBox
+
+# from PyQt5.QtWidgets import QTableWidgetItem
+
+QtCore.QCoreApplication.setSetuidAllowed(True)
+
+# from pyqtgraph import PlotWidget
+import pyqtgraph as pg
+
+import sys
+import time
+import json
+import pickle
+import socket
+
+
+IP = 10.54.5.244
```

Comment:
Is this the host IP or target IP? If host, you can probably read it dynamically with socket. Otherwise it probably needs to go in a settings file